src/pages/Main/Main.js

```diff
@@ -1,5 +1,104 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import NewProducts from './NewProducts';
+import './Main.scss';
 
 export default function Main() {
-  return <div>Main</div>;
+  const [newProducts, setNewProducts] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/newProducts.json')
+      .then(response => response.json())
+      .then(result => setNewProducts(result));
+  });
+
+  return (
+    <>
```

Comment:
```suggestion
    <Fragment className="main">
```
nesting을 위해서 가장 부모 태그에 컴포넌트 이름과 동일한 className을 넣어주셔야 하는데
fragment 태그를 사용해야 하면 위와 같은 방법으로 구현해 주시면 됩니다!